algorithms/ar-bsrh/cpp/bubble_sort.cpp

```diff
@@ -0,0 +1,42 @@
+#include <iostream>
+using namespace std;
+
+void swap(int *i, int *j) {
+	int temp = *i;
+	*i = *j;
+	*j = temp;
+}
+
+void bubbleSort(int arr[], int n) {
+	bool swapped;
```

Comment:
Codacy found an issue: [The scope of the variable 'swapped' can be reduced.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6278826)

Reply:
Thanks for your help
Please assign me too with any algorithm I will try my best to contribute.
I am not too familiar with the issue section.
Thanks